CHANGELOG.md

```diff
@@ -2,8 +2,9 @@
 
 ### 2.9.0-beta.1 
 
-> [!IMPORTANT]
+> [!Breaking Changes] 
 > [Android] When a method returns `StylePropertyValue` the property values will now be typed rather than a string.
+> `PointAnnotation.iconImageCrossFade` has been deprecated and setting a value to it will not have any impact. Use `PointAnnotationManager.iconImageCrossFadeTransition` instead.
```

Comment:
```suggestion
> `PointAnnotation.iconImageCrossFade` has been deprecated and setting a value to it will not have any impact. Use `PointAnnotationManager.iconImageCrossFade` instead.
```